tests/SyntaxParser/ImportedClassesParserTest.php

```diff
@@ -0,0 +1,126 @@
+<?php
+declare(strict_types=1);
+
+namespace Paysera\PhpCsFixerConfig\Tests\SyntaxParser;
+
+use Paysera\PhpCsFixerConfig\SyntaxParser\Entity\ImportedClasses;
+use Paysera\PhpCsFixerConfig\SyntaxParser\ImportedClassesParser;
+use PhpCsFixer\Tokenizer\Tokens;
+use PHPUnit\Framework\TestCase;
+
+class ImportedClassesParserTest extends TestCase
+{
+    /**
+     * @dataProvider parseImportedClassesFromTokensDataProvider
+     */
+    public function testParseImportedClassesFromTokens(Tokens $tokens, ImportedClasses $expectedImportedClasses): void
+    {
+        $importedClassesParser = new ImportedClassesParser();
```

Comment:
redundant var, just `$this->assertEquals($expectedImportedClasses, (new ImportedClassesParser())->parseImportedClassesFromTokens($tokens))`